test_scripts/API/DeleteCommand/ATF_DeleteCommand_VR_DeleteCommand_WARNINGS.lua

```diff
@@ -0,0 +1,191 @@
+local ServerAddress = commonFunctions.read_parameter_from_smart_device_link_ini("ServerAddress")
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,_)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+        :Times(1)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+function Test:Precondition_AddSubMenu()
+  --mobile side: sending AddSubMenu request
+  local cor_id_add_submenu = self.mobileSession:SendRPC("AddSubMenu",
+  {
+    menuID = 1000,
+    position = 0,
+    menuName ="Commandpositive"
+  })
+  --hmi side: expect UI.AddSubMenu request
+  EXPECT_HMICALL("UI.AddSubMenu", 
+  { 
+    menuID = 1000,
+    menuParams = {
+      position = 0,
+      menuName ="Commandpositive"
+    }
+  })
+  :Do(function(_,data)
+    --hmi side: sending UI.AddSubMenu response
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  end)    
+  --mobile side: expect AddSubMenu response
+  EXPECT_RESPONSE(cor_id_add_submenu, { success = true, resultCode = "SUCCESS" })
+  --mobile side: expect OnHashChange notification
+  EXPECT_NOTIFICATION("OnHashChange")
+end
+
+function Test:Precondition_AddCommand()
+  --mobile side: sending AddCommand request
+  local cid_add_cmd = self.mobileSession:SendRPC("AddCommand",
+  {
+    cmdID = 11,
+    menuParams =  
+    { 
+      parentID = 0,
+      position = 0,
+      menuName ="Commandpositive"
+    }, 
+    vrCommands = 
+    { 
+      "VRCommandonepositive",
+      "VRCommandonepositivedouble"
+    }, 
+    cmdIcon =   
+    { 
+      value ="icon.png",
+      imageType ="DYNAMIC"
+    }
+  })
+  --hmi side: expect UI.AddCommand request
+  EXPECT_HMICALL("UI.AddCommand", 
+  { 
+    cmdID = 11,
+    cmdIcon = 
+    {
+      value = storagePath.."icon.png",
```

Comment:
@vvvakulenko, please make verification of 'value' to be checked as sub-string, because at the moment the test has an error:
Exp: ./SDL_bin/storage/0000001_12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0/icon.png
Real: /home/istoimenova/SmartDeviceLinkCore/test_run/SDL_bin/storage/0000001_12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0/icon.png

You can use as example:
if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
-- print some error
end